config/install/user.role.az_content_editor.yml

```diff
@@ -41,13 +41,11 @@ permissions:
   - 'create az_course content'
   - 'create az_document media'
   - 'create az_event content'
-  - 'create az_flexible_block block content'
   - 'create az_flexible_page content'
   - 'create az_image media'
   - 'create az_news content'
   - 'create az_person content'
   - 'create az_publication content'
-  - 'create az_quick_links block content'
```

Comment:
FWIW, the new create xxxx block content permissions provided by core as of Drupal 10.1 has the same machine names so I may be adding this one back in my PR for https://github.com/az-digital/az_quickstart/issues/3014.